<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualStageControl</class>
 <widget class="QWidget" name="ManualStageControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>140</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Movement Controls&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentVelocity">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255); border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Current&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentMinEndOfRun">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255); border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetMoveLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Absolute Move (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentStagePosition">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Current Stage Position (mm)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetAcceleration">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Min End of Run (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MoveAbsolute">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="AbsoluteMoveParam">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-2</number>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NewAccelParam">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="MoveRelative">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Manual Stage Control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Current&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetMinEndOfRun">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="IncrementalMoveLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Relative Move (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Movement Configuration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Current&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>280</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Max End of Run (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NewMaxEndOfRunParam">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>65.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentMaxEndOfRun">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255); border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetMaxEndOfRun">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentAcceleration">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255); border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NewVelocityParam">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Acceleration (mm/s&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NewMinEndOfRunParam">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>65.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Velocity (mm/s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetVelocity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentPositionIndicator">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255); border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Current&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="RelativeMoveParam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>399</width>
      <height>179</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="TerminalOut">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>0</y>
       <width>411</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Terminal Output&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Movement Controls&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetPosition">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Connect To Stage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetMinEndOfRun">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetMaxEndOfRun">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetAcceleration">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetVelocity">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConnectionIndicator">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 0, 0); border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>Disconnected</string>
   </property>
  </widget>
  <widget class="QLabel" name="IncrementalMoveLabel_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Stop Motion&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopMotion">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#StopMotion{
    background-color: red;
    border-style: outset;
    border-width: 1px;
    border-color: black;
}

QPushButton#StopMotion:pressed {
    background-color: rgb(200, 0, 0);
    border-style: inset;
}

QPushButton#StopMotion:hover:!pressed {
    background-color:rgb(215,35,35);
}</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QComboBox" name="COMPortSelect">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM8</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
